### ISSUE TITLE
Idea: An option to confirm quick load

### ISSUE THREAD

**Issue by tommy61157** [PiratesAhoy/new-horizons#123]
If you hit F6, even on accident, without any confirmation, it'll just load the last save and possibly undo lots of progress if you accidentally hit it since there's no confirmation. What would be nice is if when you hit F6 for the quick load that you get a quick confirmation that you want to load the last save and warn that any unsaved data will be lost. Lost some major progress to this and would definitely appreciate this in the future. We wouldn't need this for the quick save since having an extra save doesn't hurt anyone, just quick load to ensure people don't accidentally lose progress.

**Comment by Hammie** [comment]
The ability to easily change the quickload key binding could also improve this situation a lot. (Having them next to each other is asking for trouble, and the reason most modern games put quickload on F8 or F9 instead.)

I would also make sure it is optional, as some might find the confirmation tedious.

**Comment by tommy61157** [comment]
Agreed on the rebind point, but yeah, there should at least be an option to trigger a prompt for the quick load feature, this would be so nice for those who occasionally accidentally press keys
